### PR TITLE
Delete the extra files. PEP8. Code formatting.

### PR DESCRIPTION
**Pull request by rampoe**

### CODE REVIEW

**Comment by Hojagulyyev** [comment]
Good evening, Rahman. How do you do?
I'm interested in your career achievements.
Can I ask you some questions:
- How old are you?
- Where do you study?
- Which self-development books do you read? (our interests coincide)
- Are you currently working on any project or startup?

Best regards, Hojagulyyev.

**Comment by rampoe** [comment]
> Good evening, Rahman. How do you do?

Hi! :)

> How old are you?

Well, I am 17...

> Where do you study?

I am in high school, but I also visit IT meydancha sometimes :)))

> Which self-development books do you read? (our interests coincide)

Usually it's psychology, money, time management etc. But currently I am reading "Getting Things Done" by David Allen.

> Are you currently working on any project or startup?

Yeah, I work on my small project called Growth Point :)

File: templates/_base.html
Comment (Hojagulyyev) [inline templates/_base.html:1]: may I know why you use underscore before "base"?
usually underscore is used for local variables

File: apps/users/views.py
Comment (Hojagulyyev) [inline apps/users/views.py:1]: Are you using the isort library, why is sorted import important to you?

File: apps/commits/admin.py
Comment (Hojagulyyev) [inline apps/commits/admin.py:1]: PEP 8: E303 too many blank lines (0 < x < 3)
PEP8 allows two empty lines.

**Review by Hojagulyyev — APPROVED** [review]
just a few clarifying questions

File: templates/_base.html
Comment (rampoe) [reply templates/_base.html:1]: Using an underscore before template names is a common convention to indicate that a template is intended to be used as a private (internal) template, but of course it is not a requirement in Django, so you can definitely ignore it.

File: apps/users/views.py
Comment (rampoe) [reply apps/users/views.py:1]: I use isort to automate the process of sorting imports according to PEP 8 guidelines. It improves code readability, helps in avoiding name clashes and merge conflicts.

File: apps/commits/admin.py
Comment (rampoe) [reply apps/commits/admin.py:1]: Even though PEP 8 allows two empty lines, it is recommended that only global classes and functions be separated with two empty lines.

> Surround top-level function and class definitions with two blank lines.

https://peps.python.org/pep-0008/#blank-lines

**Pull request merged**